File: MPI/mpicheck/tests/functions/2.0/MPI_Win_delete_attr.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2
        call mpi_win_delete_attr(var_0, var_1, var_2)
       call pmpi_win_delete_attr(var_0, var_1, var_2)
        end program main
    

File: MPI/mpicheck/tests/functions/2.0/MPI_Win_delete_attr.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Win var_0;
    int var_1;
    int ret;
    /* calls */
    ret = MPI_Win_delete_attr(var_0, var_1);
    ret = PMPI_Win_delete_attr(var_0, var_1);
    return 0;
}
